CHANGELOG.md

```diff
@@ -1,26 +1,268 @@
-# Release 6.0.1
+# Release 7.0.0
 
 **NOTE:**
-_This changelog only contains release notes for PWA Studio 6.0.1._
+_This changelog only contains release notes for PWA Studio 7.0.0._
 _For older release notes, see [PWA Studio releases][]._
 
 ## Table of contents
 
--   [What's new in 6.0.1](#whats-new-in-600)
+-   [What's new in 7.0.0](#whats-new-in-700)
 -   [Pull requests merged in this release](#pull-requests-merged-in-this-release)
+-   [Known issues](#known-issues)
 -   [Upgrading from a previous version](#upgrading-from-a-previous-version)
 
-## What's new in 6.0.1
+## What's new in 7.0.0
 
-PWA Studio 6.0.1 contains a hotfix for the scaffolding generator.
+PWA Studio 7.0.0 contains new features, refactors, and various improvements.
 
-This release fixes an error generated by the scaffolding generator when you run the `create-project` command directly.
+### Extensibility framework
 
-See PR [#2363](https://github.com/magento/pwa-studio/pull/2363) for additional details.
+This release improves on the extensibility framework introduced in version 6.0.0.
+It introduces new extension points for the Buildpack, Peregrine, and Venia UI library components.
+
+Developer can use these extension points to extend their storefront project without duplicating and maintaining PWA-Studio code.
+
+As of 7.0.0, PWA-Studio contains the following extension points:
+
+#### Venia UI extension points
+
+Venia UI extension points are declared in [`venia-ui-declare.js`][].
+
+| Target name            | Description                                           |
+| ---------------------- | ----------------------------------------------------- |
+| `richContentRenderers` | Add custom rich content renderers to your storefront. |
+| `routes`               | Add or modify storefront-specific routes              |
```

Comment:
I recommend either eliminating the "." at the end of each of these descriptions to reduce noise. Otherwise, let's stay consistent with including them at the end of these descriptions.